packages/models/src/settings.ts

```diff
@@ -0,0 +1,9 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+import { Tenant } from '@sap-cloud-sdk/core';
+
+export interface Settings {
```

Comment:
Same thing here, `Settings` could be anything. Consider whether more context is needed to understand this as a user.